### PR TITLE
News post about gadgetbridge support

### PR DESCRIPTION
**Pull request by eLtMosen**
Text might need some rewording and could use some extension.
But @dodoradio mentioned to have a look into that <3

grafik

### CODE REVIEW

File: pages/news/gadgetbridge-support.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Gadgetbridge AsteroidOS support
+layout: news
+posted: 2023-03-03
+post: true
+thumbnails: gadgetbridge
+summary: Gadgetbridge has added support for all AsteroidOS watches in release 0.73.0. Thanks to Noodelz for this great work on a near feature complete initial implementation!
```
Comment (dodoradio) [inline pages/news/gadgetbridge-support.md:7]: I think `feature-complete` is clearer than `feature complete` here

File: pages/news/gadgetbridge-support.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+author: Timo Könnecke (eLtMosen)
+---
+<h1>A new way to sync with your watch!</h1>
+<p><a href="https://codeberg.org/Freeyourgadget/Gadgetbridge">Gadgetbridge</a>, the application that allows you to connect your Pebble, Mi Band, Amazfit Bip and HPlus device and many more devices to your Android phone, can now sync to all AsteroidOS watches since <a href="https://codeberg.org/Freeyourgadget/Gadgetbridge/src/branch/master/CHANGELOG.md">release of version 0.73.0</a></p>
+
+<p>This has been made possible by <a href="https://codeberg.org/Freeyourgadget/Gadgetbridge/pulls/3013">Noodelz contribution to the Gadgetbridge project.</a></p>
+
+<p>Nearly all capabilities provided by AsteroidOSSync are now supported by Gadgetbridge too. Only remote volume control and the screenshot are missing.</p>
```
Comment (dodoradio) [inline pages/news/gadgetbridge-support.md:15]: I'd reword this section to 
```
Nearly all capabilities provided by AsteroidOSSync are now supported by Gadgetbridge, only the remote volume control and screenshot functions are missing.
```
I think that flows a bit better, up to you.

File: pages/news/gadgetbridge-support.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+<p>According to Noodelz, <code>"there are a few other known issues. One being that the Advertisement service is not being detected, so the only way GB knows that a device is an AsteroidOS device is by its name. I just have a list of currently supported codenames, and that works for now.<br>
+The other thing is that call support is pretty bad. If you answer a call, it doesn't dismiss the notification on the watch so it'll keep ringing. This is because calls aren't given an ID, so there's no easy way to store the ID of the call to dismiss the notification.<br>
+All updates to notifications will ring the watch. This can be good or bad. For example AntennaPod will update its download notification and send a billion updates to the watch's notification service, and each one will ring. This is because GB doesn't give whether or not a notification update is supposed to be audible or not."</code></p>
+
+<p>Below photo shows 5 asteroid watches being paired simultaniously to one phone.</p>
```
Comment (dodoradio) [inline pages/news/gadgetbridge-support.md:21]: `The below photo` over `below photo`? 

Alternatively, maybe this line should be a caption below the photo? Then it could be something like `5 Asteroid watches paired simultaneously to one phone, something not possible on AsteroidOSSync`?
once again, up to you.

File: pages/news/gadgetbridge-support.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Gadgetbridge AsteroidOS support
```
Comment (dodoradio) [inline pages/news/gadgetbridge-support.md:2]: Should this be `Gadgetbridge adds AsteroidOS support`?

**Review by dodoradio — COMMENTED** [review]
this looks really good! I've made a few comments on changes I think might improve flow or clarity, but they're all quite optional.

**Comment by eLtMosen** [comment]
Some sort of caption style added and text streamlined thanks to @dodoradio hint

grafik

File: styles/components/news.less [hunk 5]
```diff
@@ -64,7 +64,7 @@
 }
 
 .news-content p {
-    text-indent: 40px;
+    margin-left: 40px;
```
Comment (FlorentRevest) [inline styles/components/news.less:67]: Mhh... This was originally meant as a paragraph indentation https://en.wikipedia.org/wiki/Indentation_(typesetting)

But it seems to annoy you so let's just get rid of the margin entirely.

**Review by FlorentRevest — APPROVED** [review]
Great :) Feel free to submit once the css thing is addressed.

File: styles/components/news.less (hunk 5 above)
Comment (eLtMosen) [reply styles/components/news.less:67]: The intendation is great when used on multi line text. But our page is one <p> after another and tus also single lines getting 40px indented. Which results in only on line of "real" left aligned floating text. looking very awkward. Like lists that are broken and do not indent the next line.
I am going to redesign to multiple columns. and maybe paragraph indents will make sense when the floating text is better consolidated into fewer paragraphs. Lets see.

**Comment by FlorentRevest** [comment]
Ugh, sorry to annoy you! In the case of an article, I prefer the one column style.

And I understand that the paragraph indentation looks somewhat weird, I'm happy if we just remove it.

Also, another thing that crossed my mind is, could you write this article in markdown ? (you'd still be able to use HTML for the outlier cases that are not handled by the Markdown syntax). The problem is that people will copy your article and if we're trying to set a precedent on how writing articles is an easy process, we should show the easy way (or people will more easily get discouraged in the future)

File: pages/news/gadgetbridge-support.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+layout: news
+posted: 2023-03-03
+post: true
+thumbnails: gadgetbridge
+summary: Gadgetbridge has added support for all AsteroidOS watches in release 0.73.0. Thanks to Noodelz for this great work on a near feature-complete initial implementation!
+author: Timo Könnecke (eLtMosen)
+---
+<h1 class="news-header-centered">A new way to sync with your watch!</h1>
+<div class="news-two-column">
+<div class="news-two-col-paragraph"><a href="https://codeberg.org/Freeyourgadget/Gadgetbridge">Gadgetbridge</a>, the application that allows you to connect your Pebble, Mi Band, Amazfit Bip and HPlus device and many more devices to your Android phone, can now sync to all AsteroidOS watches since <a href="https://codeberg.org/Freeyourgadget/Gadgetbridge/src/branch/master/CHANGELOG.md">release of version 0.73.0</a></div>
```
Comment (FlorentRevest) [inline pages/news/gadgetbridge-support.md:12]: I think we can make this paragraph a tiny bit easier to read (and mention that Gadgetbridge is open-source) with something like

Gadgetbridge, the open-source Android application that supports connection to Pebble, Mi Band, Amazfit Bip devices and many devices can now also sync to AsteroidOS watches since v0.73.

**Comment by FlorentRevest** [comment]
@beroset pointed out that Markdown doesn't actually work yet in the news. We would need to fix our news template but I don't want to overload you. Forget about Markdown for now...

File: pages/news/gadgetbridge-support.md (hunk 6 above)
Comment (eLtMosen) [reply pages/news/gadgetbridge-support.md]: Sounds better indeed!

File: pages/news/gadgetbridge-support.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+author: Timo Könnecke (eLtMosen)
+---
+# A new way to sync with your watch!
+[Gadgetbridge](https://codeberg.org/Freeyourgadget/Gadgetbridge), the open-source Android application that supports connection to Pebble, Mi Band, Amazfit Bip devices and many devices can now also sync to AsteroidOS watches since [release of version 0.73.0](https://codeberg.org/Freeyourgadget/Gadgetbridge/src/branch/master/CHANGELOG.md)
+
+This has been made possible by [Noodelz contribution to the Gadgetbridge project.](https://codeberg.org/Freeyourgadget/Gadgetbridge/pulls/3013)
+Nearly all capabilities provided by AsteroidOSSync are now supported by Gadgetbridge, only the remote volume control and screenshot functions are missing.
+Contributions to further polish the Gadgetbridge integration are highly appreciated. Visit our [Matrix channel](https://matrix.to/#/#Asteroid:matrix.org) for help on how to get started.
+
+According to Noodelz, `"there are a few other known issues. One being that the Advertisement service is not being detected, so the only way GB knows that a device is an AsteroidOS device is by its name. I just have a list of currently supported codenames, and that works for now.
```
[the remark this answers is not in the document]
Comment (eLtMosen) [reply pages/news/gadgetbridge-support.md]: Good idea, implemented!

**Review by beroset — APPROVED** [review]
Looks good!

File: pages/news/gadgetbridge-support.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+The other thing is that call support is pretty bad. If you answer a call, it doesn't dismiss the notification on the watch so it'll keep ringing. This is because calls aren't given an ID, so there's no easy way to store the ID of the call to dismiss the notification.
+All updates to notifications will ring the watch. This can be good or bad. For example AntennaPod will update its download notification and send a billion updates to the watch's notification service, and each one will ring. This is because GB doesn't give whether or not a notification update is supposed to be audible or not."`
+
+<img style="width: 60%; height: auto; margin: 0 15VW 0 15VW" src="/public/img/news-img/gadgetbridge-running.jpg" />
```
[the remark this answers is not in the document]
Comment (eLtMosen) [reply pages/news/gadgetbridge-support.md]: Agreed, 600px (vs 900px) height should be enough. Its 24kb now converted to webp.
Mind however, that the 60% that where set in css/style tag are the viewport width. Making the image responsive.
Only scaling the image file is not enough to ensure that the image is not too large on phones and produces horizontal scroll.
I added a news-img-centered class to offload the css.

**Comment by FlorentRevest** [comment]
Thanks a lot! Sorry it took so long :D

**Comment by eLtMosen** [comment]
Hu, no its all right. We are finding a new format for the news afterall.

**Review by beroset — COMMENTED** [review]
Do we want every news picture to be 60% too big?  Why not just scale the picture?
